docs/api/input.md

```diff
@@ -134,6 +134,14 @@ import Migration from '@site/static/usage/v7/input/migration/index.md';
 
 Ionic uses heuristics to detect if an app is using the modern input syntax. In some instances, it may be preferable to continue using the legacy syntax. Developers can set the `legacy` property on `ion-input` to `true` to force that instance of the input to use the legacy syntax.
 
+## Input Masking
+
+Input masks are expressions that constrain input to support valid input values. Ionic recommends using [Maskito](https://tinkoff.github.io/maskito/getting-started/what-is-maskito) for input masking. Maskito is a lightweight, dependency-free library for masking input fields. It supports a wide range of masks, including phone numbers, credit cards, dates, and more.
```

Comment:
Can maskito be used with ion-textarea and ion-searchbar? If so, maybe we should note that too.

Reply:
Thoughts on having separate tickets for creating playground examples for those components? I haven't extensively tested input masking with those components and figured we would start with `ion-input` and then continue to the other controls. 

In theory they should work though.

Reply:
yeah that makes sense